src/common/commonutils/OtherUtils.c

```diff
@@ -41,13 +30,12 @@ char* DuplicateStringToLowercase(const char* source)
     return duplicate;
 }
 
-#define DEFAULT_FORMAT_ALLOCATE_STRING_LENGTH 512
 char* FormatAllocateString(const char* format, ...)
 {
     char* buffer = NULL;
     char* stringToReturn = NULL;
     int formatResult = 0;
-    int sizeOfBuffer = DEFAULT_FORMAT_ALLOCATE_STRING_LENGTH;
+    int sizeOfBuffer = MAX_STRING_LENGTH;

```

Comment:
Is there any chance this is used for banners, etc. that would require a larger buffer than specified? I assume not.

Reply:
good point SSH login screens tend to be quite large

Reply:
If you mean the SSH banner, that one is s static string, coming as an Azure Policy parameter. Not dynamic. We may increase this size in the future if/when we will need to format larger strings.